humanistica-lovaniensia-journal-of-neo-latin-studies.csl

```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only">
+  <!-- Polyglot; journal publishes in English, French, German, Italian and Spanish. -->
+  <info>
+    <title>Humanistica Lovaniensia. Journal of Neo-Latin Studies</title>
```

Comment:
@rmzelle Opinion on the subtitle? The journal consistently uses it & uses it with a period. In the US (e.g. in JSTOR) it's listed with a colon. I think any of the three possible options (period, colon, no subtitle) is fine, not sure we have super clear rules on this.

Reply:
> I think any of the three possible options (period, colon, no subtitle) is fine, not sure we have super clear rules on this.

We never standardized on this, so I suggest to just follow whatever the journal uses.

Reply:
(I do prefer replacing all dashes with hyphens)